src/components/ContentSwitch/ResponsiveSwitchPanel.part.tsx

```diff
@@ -56,9 +56,10 @@ class ContentSwitchInt extends React.Component<ContentSwitchProps, ContentSwitch
 
     if (headers && container && orientation === 'horizontal') {
       if (!headerNodesWidth.length) {
-        headers.childNodes.forEach((node: HTMLLIElement) => {
```

Comment:
Can you try `[...headers.childNodes].forEach(...)` instead of rewriting it to for/of?

Reply:
Used "Array.from" instead of "for-of."